HISTORY.md

```diff
@@ -1,3 +1,8 @@
+3.14.0 / 2018-03-23
```

Comment:
Could you change this header to "UNRELEASED"?

Reply:
Sure! I will change this right away.